### DISCUSSION TITLE
Missing channel (#ukranian)

### DISCUSSION THREAD

**Discussion by CX13BVA** [ppy/osu#21976] [Q&A] [answered]
#ukranian can't be found in the lazer client.

![](http://puu.sh/JvsxQ/9789c9b24f.jpg)

https://osu.ppy.sh/community/forums/topics/1699578?n=1

**Comment by bdach** [comment, marked as answer]
As correctly noted in the forum thread this is an osu-web consideration. To that end: https://github.com/ppy/osu-web/issues/9700

**Reply by peppy** [reply]
Has been resolved now.
